object_detection/README.md

```diff
@@ -0,0 +1,56 @@
+# Object Detection
+
+This recipe provides an example for running an object detection model service and its associated client locally. 
+
+## Build and run the model service
+
+```bash
+podman build -t object_detection_service object_detection/model_server 
+```
+
+```bash
+podman run -it --rm -p 8000:8000 object_detection_service
+```
+
+By default the model service will use [`facebook/detr-resnet-101`](https://huggingface.co/facebook/detr-resnet-101), which has an apache-2.0 license. The model is relatively small, but it will be downloaded fresh each time the model server is started unless a local model is provided (see additional instructions below).  
+
+
+## Use a different or local model
+
+If you'd like to use a different model hosted on huggingface, simply use the environment variable `MODEL_PATH` and set it to the correct `org/model` path on [huggingface.co](https://huggingface.co/) when starting your container. 
+
+If you'd like to download models locally so that they are not pulled each time the container restarts, you can use the following python snippet to a model to your `models/` directory.  
+
+```python
+from huggingface_hub import snapshot_download
+
+snapshot_download(repo_id="facebook/detr-resnet-101",
+                  revision="no_timm",
+                local_dir="<PATH_TO>/locallm/models/vision/object_detection/facebook/detr-resnet-101",
+                local_dir_use_symlinks=False)
+
+```
+
+When using a model other than the default, you will need to set the `MODEL_PATH` environment variable. Here is an example of running the model service with a local model:
+
+```bash
+ podman run -it --rm -p 8000:8000 -v <PATH/TO>/locallm/models/vision/:/locallm/models -e MODEL_PATH=models/object_detection/facebook/detr-resnet-50/ object_detection_service
+```
+
+## Build and run the client application
+
+```bash
+podman build -t object_detection_client object_detection/model_client 
+```
+
+```bash
+podman run -p 8501:8501 -e MODEL_ENDPOINT=http://10.88.0.1:8000/detection object_detection_client
```

Comment:
I think we should give instruction how to retrieve the IP address of the model service container or have them run in a pod

Reply:
Can we add that as a separate issue that we can address in another PR and not block on it here? All the recipe docs currently reference 10.88.0.1. So they will all need to be changed accordingly.